README.md

```diff
@@ -117,12 +117,10 @@ Create Mapbox Vector Tiles from raster sources
 
 ## Implementations
 
-#### [**rio-viz**][rio-viz]
+#### [**rio-viz**](https://github.com/developmentseed/rio-viz)
 
 ![](https://user-images.githubusercontent.com/10407788/105772356-0ca2d900-5f30-11eb-85b9-c3da9e12b663.jpg)
 
-[rio-viz]: (https://github.com/developmentseed/rio-viz)
```

Comment:
Oh this isn't supposed to be in parentheses. I'd suggest just removing the parentheses here instead of making an inline link.

Reply:
That also works :)